ui/Screen/Settings.svelte

```diff
@@ -197,7 +198,7 @@
           <h3>Devices</h3>
         </header>
         <div class="section-item">
-          <div class="info">
+          <div>
```

Comment:
I think we can remove all this unnecessary divs that had the `info` class before.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/2038/commits/bda1aa5202029dfd406298f6c9913e04d2fbb754

Reply:
Removed them where I could